src/odemis/gui/cont/tabs.py

```diff
@@ -2825,9 +2833,34 @@ def _onSEMpxs(self, pixel_size):
         best_mpp = self._sem_view.mpp.clip(best_mpp)
         self._sem_view.mpp.value = best_mpp
 
+    def _onAlignPos(self, pos):
+        """
+        Called when the aligner is moved (and the tab is shown)
+        :param pos: (dict str->float or None) updated position of the aligner
+        """
+        # Return if aligner doesn't have both MD_FAV_POS_ACTIVE and MD_FAV_POS_DEACTIVE metadata
```

Comment:
Please add a check that the tab is currently shown. That's an extra safety, mostly in case the user moved to the tab, and immediately left to another tab...  while later the aligner goes to "active" position, and subscribes to this.